### ISSUE TITLE
Interop: Applying system accessibility preferences to WebVTT cues

### ISSUE THREAD

**Issue by jernoble** [w3c/webvtt#536]
Safari and Chrome (on macOS) read system accessibility preferences and convert the user's preferred caption settings into CSS styles. These styles are then applied to the WebVTT cues rendered by the UA.  Attached are examples of Chrome and Safari rendering with various accessibility preferences set.

Firefox (on macOS, at least) does not read system accessibility preferences.

Additionally, Safari and Chrome implement WebVTT rendering within the media element Shadow DOM, which is closed, and therefore inaccessible from the rest of the page. Queries of the type `getComputedStyle(video, "::cue")` fail to apply styles defined inside the Shadow DOM, which includes both system accessibility derived styles, as well as styles applied by the WebVTT file itself.

Concerns have been [raised](https://github.com/w3c/webvtt/pull/528#issuecomment-3525528637) in PR #520 that UAs which apply users' system accessibility preferences to WebVTT rendering may cause fingerprinting issues due to exposing those users' distinguishing preferences to web content. Safari and Chrome's implementation details are intended to prevent that exposure, but they also introduce an interop issue.

### Questions:

* Should the specification include a note regarding system accessibility preferences?
* Should the specification suggest or require those preferences to be hidden from the page?  

### Examples:

Default Caption Settings, Chrome Rendering, Safari Rendering:
<img width="474" height="260" alt="Default Caption Settings" src="https://github.com/user-attachments/assets/0f5a34ff-dc0b-4cd0-8e16-e6a3f7ceb84f" /><img width="320" height="295" alt="Chrome Default Caption Rendering" src="https://github.com/user-attachments/assets/354ac720-e1de-4ba0-89c0-e872005ed7a0" /><img width="320" height="280" alt="Safari Default Caption Rendering" src="https://github.com/user-attachments/assets/73fff8f8-4a09-4eff-ba3b-0645141bed55" />

Classic Caption Settings, Chrome Rendering, Safari Rendering:
<img width="474" height="260" alt="Classic Caption Settings" src="https://github.com/user-attachments/assets/da924c82-1fe6-44b1-ac95-99fd6014b743" />
Chrome Classic Caption Rendering:
<img width="320" height="295" alt="Chrome Classic Caption Rendering" src="https://github.com/user-attachments/assets/42e6074c-6ec4-4724-b4f4-03e715007782" />
Safari Classic Caption Rendering:
<img width="320" height="277" alt="Safari Classic Caption Rendering" src="https://github.com/user-attachments/assets/c60ec189-a729-41e0-9f99-429773036c19" />

Custom Caption Settings, Chrome Rendering, Safari Rendering:
<img width="474" height="260" alt="Custom Caption Settings" src="https://github.com/user-attachments/assets/7f219a72-691e-4dfe-81fe-57219edbb830" /><img width="320" height="295" alt="Chrome Custom Caption Rendering" src="https://github.com/user-attachments/assets/5a1d0f47-6b95-4643-9ebf-a93af48ab8a9" /><img width="320" height="277" alt="Safari Custom Caption Rendering" src="https://github.com/user-attachments/assets/c6a4118f-b1a0-43a9-92cf-a131bea9fc79" />
